examples/plot_knockoff_aggregation.py

```diff
@@ -167,12 +153,81 @@ def plot_results(bounds, fdr, nsubjects, n_clusters, rho, power=False):
             rotation=45,
             ha="right",
         )
-        plt.title(f"FDR = {fdr}, n = {nsubjects}, p = {n_clusters}, rho = {rho}")
+        plt.title(f"FDR = {fdr}, n = {n_samples}, p = {n_features}")
         plt.ylabel("Empirical FDP")
         plt.legend(loc="best")
 
+
+#######################################################################
+# Define the function for evaluate the effect of the population
+# -------------------------------------------------------------
+def effect_number_samples(n_samples, sparsity=sparsity):
+    parallel = Parallel(n_jobs, verbose=joblib_verbose)
+    results = parallel(
+        delayed(single_run)(
+            n_samples, n_features, rho, sparsity, snr, fdr, n_bootstraps, seed=seed
+        )
+        for seed in seed_list
+    )
+
+    fdps_mx = []
+    fdps_pval = []
+    fdps_eval = []
+    powers_mx = []
+    powers_pval = []
+    powers_eval = []
+    for fdp_mx, fdp_pval, fdp_eval, power_mx, power_pval, power_eval in results:
+        fdps_mx.append(fdp_mx)
+        fdps_pval.append(fdp_pval)
+        fdps_eval.append(fdp_eval)
+
+        powers_mx.append(power_mx)
+        powers_pval.append(power_pval)
+        powers_eval.append(power_eval)
+
+    # Plot FDP and Power distributions
+
+    fdps = [fdps_mx, fdps_pval, fdps_eval]
+    powers = [powers_mx, powers_pval, powers_eval]
+
+    plot_results(fdps, fdr, n_samples, n_features)
+    plot_results(powers, fdr, n_samples, n_features, power=True)
     plt.show()
 
 
-plot_results(fdps, fdr, n_subjects, n_clusters, rho)
-plot_results(powers, fdr, n_subjects, n_clusters, rho, power=True)
+#######################################################################
+# Aggregation methods provide a more stable inference
+# ---------------------------------------------------
+effect_number_samples(n_samples=n_samples)
+
+#######################################################################
+# By repeating the model-X Knockoffs, we can see that instability
+# of the inference. Additionally, we can see that the aggregation method
+# is more stable. However, the e-values aggregation is more conservative,
+# i.e. the exepect variables of importance is not find.
+
+#######################################################################
+# Aggregation methods depends on your data
+# ----------------------------------------
+effect_number_samples(n_samples=n_samples, sparsity=0.1)
```

Comment:
Please don't change the sparsity here. It breaks the power (for good reasons).

Reply:
It's specific to the knockoffs framework: given the knockoffs decision function, in order to be able to detect *something* you need to detect at least  1 / fdr true positive detections (which means 10 true detections for fdr=.1). 
So,  if the ground truth is very sparse (e.g., sparsity of .1 for p=100 variables means only 10 truly predictive variables) it becomes super challenging: if you only miss one variables in true support, you're dead.  
With a less stringent  sparsity, e.g. 0.2, the true positive set comprises 20 variables, which makes it more likely that you will detect 10 among them.
Does that make sense ?

Reply:
I think that this can be interesting to show it in an example.
Do you want to explain here or to show it in another example?

Reply:
I'd rather have that in another example.